README.md

```diff
@@ -1 +1,5 @@
-# codereview-1
\ No newline at end of file
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```

Comment:
- [ ] Update your README to have a description of the project to best guide visitors to the repo. Use this [template](https://github.com/microverseinc/readme-template) for reference